Issue #8895 - Introduce `jetty-home-<ver>-with-docs.zip`

joakime opened the pull request:
Experiment to see if a `jetty-home-<ver>-with-docs.zip` can be done, and makes sense.

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime committed:
Issue #8895 - Introduce jetty-home-<ver>-with-docs.zip

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime commented:
This isn't 100% ready (yet).
Still have some testing to do, and some cleanup.
There's assumptions made in here that need validation as well.

joakime committed:
Unpacking the right javadoc artifact

joakime commented:
The output ...

```
[jetty.project-10.0.x][fix/jetty-10-home-with-docs]$ ls -la jetty-home/target/*.zip
-rw-rw-r-- 1 joakim joakim 11258702 Nov 14 15:45 jetty-home/target/jetty-home-10.0.13-SNAPSHOT-sources.zip
-rw-rw-r-- 1 joakim joakim 36232233 Nov 14 15:45 jetty-home/target/jetty-home-10.0.13-SNAPSHOT-with-docs.zip
-rw-rw-r-- 1 joakim joakim 23129503 Nov 14 15:45 jetty-home/target/jetty-home-10.0.13-SNAPSHOT.zip

[jetty.project-10.0.x][fix/jetty-10-home-with-docs]$ unzip -l jetty-home/target/jetty-home-10.0.13-SNAPSHOT-with-docs.zip | grep -E "/$"
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/http2/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/annotations/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/jaspi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/setuid/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/websocket/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/http3/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/apache-jsp/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/logging/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/fcgi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/lib/apache-jstl/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/gcloud/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/jmx.d/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/conscrypt/
        0  2022-11-14 15:44   jetty-home-10.0.13-SNAPSHOT/modules/sessions/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/sessions/jdbc/
        0  2022-11-14 15:44   jetty-home-10.0.13-SNAPSHOT/modules/sessions/infinispan/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/sessions/infinispan/remote/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/sessions/infinispan/remote/resources/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/sessions/infinispan/embedded/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/sessions/mongo/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/sessions/session-data-cache/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/jetty-quickstart.d/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/demo.d/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/demo.d/root/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/demo.d/root/images/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/rewrite/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/jolokia/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/log4j1/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/log4j1/resources/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/log4j1/etc/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/logback/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/logback/resources/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/logback/etc/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/jul/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/jul/resources/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/jetty/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/jetty/resources/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/log4j2/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/log4j2/resources/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/logging/log4j2/etc/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/openid/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/inetaccess/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/global-webapp-common.d/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/hawtio/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/modules/jamon/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/bin/
        0  2022-11-14 15:44   jetty-home-10.0.13-SNAPSHOT/etc/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/cdi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/jaspi/
        0  2022-11-14 15:44   jetty-home-10.0.13-SNAPSHOT/etc/sessions/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/sessions/gcloud/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/sessions/jdbc/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/sessions/hazelcast/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/sessions/infinispan/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/sessions/file/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/sessions/mongo/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/etc/sessions/session-data-cache/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/META-INF/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/META-INF/maven/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/META-INF/maven/org.eclipse.jetty.documentation/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/META-INF/maven/org.eclipse.jetty.documentation/jetty-documentation/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/client/http2/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/client/websocket/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/client/http/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/client/http3/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/server/http2/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/server/session/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/server/websocket/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/server/http/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/docs/org/eclipse/jetty/docs/programming/server/http3/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/script-dir/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/resources/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/legal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/META-INF/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/gcloud/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/gcloud/session/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/gcloud/session/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/generator/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/generator/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/frames/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/frames/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/hpack/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/hpack/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/client/http/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/client/http/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/api/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/api/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/api/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/api/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/parser/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http2/parser/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/session/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/session/infinispan/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/session/infinispan/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/proxy/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/proxy/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/preventers/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/preventers/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/annotation/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/annotation/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/resource/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/resource/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/thread/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/thread/strategy/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/thread/strategy/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/thread/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/ajax/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/ajax/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/log/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/log/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/component/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/component/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/ssl/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/ssl/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/compression/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/compression/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/security/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/security/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/statistic/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/util/statistic/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/cdi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/cdi/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/annotations/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/annotations/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/annotations/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/annotations/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/utils/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/utils/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/utils/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/utils/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/internal/serverfactory/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/internal/serverfactory/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/internal/webapp/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/internal/webapp/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/osgi/boot/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/annotation/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/annotation/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/jndi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/jndi/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/security/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/security/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/webapp/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/plus/webapp/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jspc/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jspc/plugin/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jspc/plugin/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jsp/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jsp/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/util/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/util/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/dynamic/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/dynamic/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/http/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/http/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/api/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/api/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/unixsocket/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/unixsocket/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/unixsocket/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/unixsocket/common/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/unixsocket/common/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/unixsocket/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/unixsocket/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/runner/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/runner/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/conscrypt/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/conscrypt/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/conscrypt/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/conscrypt/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/conscrypt/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/java/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/java/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/java/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/java/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/java/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/alpn/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/rewrite/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/rewrite/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/rewrite/handler/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/rewrite/handler/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/graph/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/graph/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/util/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/util/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/providers/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/providers/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/providers/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/providers/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/bindings/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/bindings/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/deploy/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/factories/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/factories/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/local/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/local/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/java/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/java/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jndi/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/client/config/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/client/config/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/client/impl/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/client/impl/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/client/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/client/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/internal/messages/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/internal/messages/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/internal/util/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/internal/util/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/exception/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/exception/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/server/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/server/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/core/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/util/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/util/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/annotations/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/annotations/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/exceptions/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/exceptions/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/api/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/servlet/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/servlet/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/common/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/common/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/server/config/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/server/config/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/server/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/server/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/client/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/client/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/messages/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/messages/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/decoders/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/decoders/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/encoders/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/encoders/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/common/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/server/config/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/server/config/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/server/internal/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/websocket/javax/server/internal/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http/pathmap/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http/pathmap/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http/spi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http/spi/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/frames/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/frames/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/client/http/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/client/http/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/api/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/api/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/qpack/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/http3/qpack/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/memcached/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/memcached/session/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/memcached/session/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quic/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quic/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quic/client/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quic/common/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quic/common/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quic/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quic/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/servlet/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/servlet/listener/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/servlet/listener/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/servlet/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/servlet/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/servlet/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/io/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/io/ssl/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/io/ssl/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/io/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/io/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/io/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quickstart/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/quickstart/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/modules/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/modules/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/provider/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/provider/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/callback/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/jaspi/callback/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/openid/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/openid/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/authentication/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/security/authentication/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/logging/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/logging/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/fileinits/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/fileinits/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/config/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/config/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/builders/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/builders/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/start/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/session/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/session/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/resource/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/resource/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/handler/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/handler/gzip/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/handler/gzip/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/handler/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/handler/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/server/handler/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/nosql/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/nosql/mongodb/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/nosql/mongodb/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/nosql/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jmx/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jmx/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jaas/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jaas/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jaas/spi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jaas/spi/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jaas/callback/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/jaas/callback/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/maven/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/maven/plugin/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/maven/plugin/utils/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/maven/plugin/utils/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/maven/plugin/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/webapp/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/webapp/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/generator/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/generator/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/client/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/client/http/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/client/http/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/server/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/server/proxy/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/server/proxy/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/server/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/parser/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/fcgi/parser/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/ant/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/ant/utils/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/ant/utils/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/ant/types/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/ant/types/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/ant/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/apache/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/apache/jsp/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/apache/jsp/class-use/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/xml/
        0  2022-11-14 15:45   jetty-home-10.0.13-SNAPSHOT/javadoc/org/eclipse/jetty/xml/class-use/


```

jmcc0nn3ll commented:
Doesn't lengthen the build appreciably, seems fine.

jmcc0nn3ll reviewed (commented):
We probably want to link to these from the demo page on the distribution, but looks good.

olamy commented on jetty-home/pom.xml:
```diff
@@ -876,6 +925,14 @@
+      <classifier>javadoc</classifier>
+      <type>jar</type>
+      <optional>true</optional>
+    </dependency>
```
you need to add jetty-documentation as a dependency as well to be sure Maven will calculate it as part of the build plan reactor.
Just run `mvn clean install -pl :jetty-home -am` and you will see it's not build before building `jetty-home` whereas it's needed.

olamy commented:
sounds good.

joakime committed:
Issue #8895 - Fixing jetty-documentation dep reference

Now the jetty-home artifact builds at the right time in the reactor.

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime committed:
Issue #8895 - Move JettyHomeTester to own module

+ Fix for circular dependency loop
    test-distribution ->
    jetty-home ->
    jetty-documentation ->
    jetty-asciidoctor-extensions ->
    test-distribution
+ Remove any deps on anything in org.eclipse.jetty from new module

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime committed:
Issue #8895 - fixing JettyHomeTester.await logs timeout

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

olamy commented on tests/jetty-home-tester/src/main/java/org/eclipse/jetty/tests/hometester/JettyHomeTester.java:
```diff
@@ -587,8 +586,8 @@ public boolean awaitLogsFileFor(Path logFile, String txt, long time, TimeUnit un
             {
-                long start = NanoTime.now();
-                while (NanoTime.since(start) < unit.toNanos(time))
+                long end = System.currentTimeMillis() + unit.toMillis(time);
```
/me thinking loud :)
I always thought we could have very much simple code here by using this very convenient tool https://github.com/awaitility/awaitility

olamy commented:
doh looks to be some chicken and eggs problem to solve here :)

olamy committed:
use Awaitility

Signed-off-by: Olivier Lamy <olamy@apache.org>

olamy committed:
review comments

joakime committed:
Issue #8895 - More reliable JettyHomeTester.unzip

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime committed:
Merge remote-tracking branch 'origin/fix/jetty-10-home-with-docs-awaitility' into fix/jetty-10-home-with-docs

# Conflicts:
#	tests/jetty-home-tester/src/main/java/org/eclipse/jetty/tests/hometester/JettyHomeTester.java

joakime committed:
Cleanup imports

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime commented on jetty-maven-plugin/pom.xml:
```diff
@@ -252,14 +252,18 @@
-    <dependency>
+    <!--<dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-home</artifactId>
```
@olamy do you recall why is this dependency here?
When I comment it out, the plugin still works (and tests properly).

joakime replied:
I'll just delete it.

olamy replied:
Because jetty-maven-plugin depends on it (see https://github.com/eclipse/jetty.project/blob/3569f6506618f0a4d95f76b72979cc0548cc6a42/jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/AbstractWebAppMojo.java#L68 and https://github.com/eclipse/jetty.project/blob/3569f6506618f0a4d95f76b72979cc0548cc6a42/jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/AbstractWebAppMojo.java#L557) there is forked mode using/downloaded the distro
It can work because you download a SNAPSHOT of jetty-home or use one already here locally but it really depends on it.
Try running: `mvn clean install -pl :jetty-maven-plugin -am` with an empty repository.

joakime replied:
I can see that `jetty-home` doesn't build before `jetty-maven-plugin`, but I really cannot make it fail, even with a new `<version>`!

joakime committed:
Remove jetty-home dependency from jetty-maven-plugin

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime committed:
Correct osgi bundle-name

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime committed:
Fixing proxyTo url for javadoc

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

joakime committed:
Referencing included /docs/ and /javadoc/ in root index.html

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

olamy commented on jetty-maven-plugin/pom.xml:
```diff
@@ -252,120 +252,6 @@
       <artifactId>jetty-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```
nope it depends on it. 
see comment here https://github.com/eclipse/jetty.project/pull/8898#discussion_r1024496941

joakime replied:
Yet the build is green without it present.

joakime replied:
With this present, we get ...

```
[ERROR] [ERROR] The projects in the reactor contain a cyclic reference: Edge between
   'Vertex{label='org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT'}' and 
   'Vertex{label='org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT'}' introduces to cycle in the graph
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-home:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT @ 
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 
   'Vertex{label='org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT'}' and 
   'Vertex{label='org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT'}' introduces to cycle in the graph
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-home:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT -> [Help 1]
```

Perhaps we should remove jetty-maven-plugin from the javadoc?

joakime committed:
Remove maven-plugins from javadoc

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

olamy commented:
this cause https://github.com/eclipse/jetty.project/issues/10312
